AISKU/scripts/listCdnVersions.ps1

```diff
@@ -373,19 +379,28 @@ Function ValidateAccess
 Function GetVersion(
     [string] $name
 ) {
-    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*(-[^\.]+)?)(\.(?:gbl\.js|gbl\.min\.js|cjs\.js|cjs\.min\.js|js|min\.js)(?:\.map)?)$'
+    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*(-[^\.]+)?)(\.(?:gbl\.js|gbl\.min\.js|cjs\.js|cjs\.min\.js|js|min\.js|integrity\.json)(?:\.map)?)$'
     $match = ($name | select-string $regMatch -AllMatches).matches
+    $contentType = $jsContentType
 
     if ($null -eq $match) {
         return $null
     }
+
+    $ext = $match.groups[6].value
+    $tokens = $ext.split(".")
+    if ($tokens.length -gt 0) {
```

Comment:
what does `-gt` mean?

Reply:
-gt is powershells version of greater then ```>```